independent-publisher-connectors/Airtable/apiDefinition.swagger.json

```diff
@@ -0,0 +1,490 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Airtable",
+    "description": "Airtable custom connector to read and write data",
+    "version": "1.0",
+    "contact": {
+      "name": "Woong Choi",
+      "email": "Woong.Choi@sevensigma.com.au"
```

Comment:
@last72 Can you please provide email address as well.

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
The email address is in the code (Line 9). Do I need to put non-work email address?

Reply:
@last72 Oh Sorry, I meant `website` of your support team.